examples/glasplaat.json

```diff
@@ -0,0 +1,171 @@
+{
+  "@context": [
+    "http://iiif.io/api/presentation/3/context.json"
+  ],
+  "id": "https://raw.githubusercontent.com/viaacode/iiif-manifests/IIIF-52-simple-example/examples/glasplaat.json",
+  "type": "Manifest",
+  "behavior": [
+    "individuals"
+  ],
+  "label": {
+    "nl": [
+      "Vlakbeelden Midden Rijk"
+    ]
+  },
+  "metadata": [
+    {
+      "label": {
+        "en": [
+          "Type"
+        ]
+      },
+      "value": {
+        "en": [
+          "Photo"
+        ]
+      }
+    },
+    {
+      "label": {
+        "en": [
+          "Date"
+        ]
+      },
+      "value": {
+        "none": [
+          "1900/1960 <dcterms:created^edtf>"
+        ]
+      }
+    },
+    {
+      "label": {
+        "en": [
+          "pid"
+        ]
+      },
+      "value": {
+        "none": [
+          "5d8nc8tp93"
+        ]
+      }
+    },
+    {
+      "label": {
+        "en": [
+          "Identifiers"
+        ]
+      },
+      "value": {
+        "none": [
+          "BIB.GLAS.016209"
+        ]
+      }
+    }
+  ],
+  "thumbnail": [
+    {
+      "id": "https://images.meemoo.be/iiif/public/OR-0z70w1b/f6/f64922b8a6314067a1a928119060f51659de2ef8245246e3b8676111c9adc34b9bf8c4f666c14a41936dbeb98669fe4d/full/480,/0/default.jpg",
+      "type": "Image",
+      "format": "image/jpeg",
+      "service": [
+        {
+          "id": "https://images.meemoo.be/iiif/public/OR-0z70w1b/f6/f64922b8a6314067a1a928119060f51659de2ef8245246e3b8676111c9adc34b9bf8c4f666c14a41936dbeb98669fe4d",
+          "protocol": "http://iiif.io/api/image",
+          "profile": "level1"
+        }
+      ]
+    }
+  ],
+  "requiredStatement": {
+    "label": {
+      "en": [
+        "Attribution"
+      ],
+      "nl": [
+        "Attributie"
+      ]
+    },
+    "value": {
+      "none": [
+        "Aangeboden door UGent"
+      ]
+    }
+  },
+  "rights": "<schema:license> Er is geen CC of Public Domain license toegekend",
```

Comment:
dit klopt niet volgens de spec. die zegt: _The value must be drawn from the set of Creative Commons license URIs, the RightsStatements.org rights statement URIs, or those added via the extension mechanism. The inclusion of this property is informative, and for example could be used to display an icon representing the rights assertions._ 

Andere rechteninformatie kan wel nog onder `metadata` toegevoegd worden.

Ik vermoed ook dat die `<schema:license>` hier niet moet staan?

Reply:
Dit is een placeholder. Hier zou de CC license moeten komen (indien die is ingevuld in het MAM). Ik zal er een voorbeeld zetten zodat dit duidelijker is.